alembic/env.py

```diff
@@ -0,0 +1,80 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+from temperatures import models
+
+target_metadata = models.Base.metadata
+
+target_metadata = None
```

Comment:
The `target_metadata` is initially set to `models.Base.metadata`, which is correct for autogeneration of migrations. However, it is immediately overwritten to `None` on line 23. This will prevent Alembic from being able to autogenerate migrations based on your models. You should remove or correct the line setting `target_metadata` to `None`.